FaunaDB.Client/Types/Api21/RecursiveRef.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using FaunaDB.Query;
+using Newtonsoft.Json;
+
+namespace FaunaDB.Types.Api21
+{
+    public abstract class ObjectRefV : RefV
```

Comment:
yes, the base class cannot be less accessible than its derived

`Inconsistent accessibility: base class FaunaDB.Types.Api21.ObjectRefV is less accessible than class FaunaDB.Types.Api21.DatabaseV`